src/site/content/en/patterns/web-vitals-patterns/infinite-scroll/infinite-scroll/assets/body.html

```diff
@@ -7,6 +7,6 @@
     <div class="item">E</div>
     <button id="infinite-scroll-button" disabled>
         <span class="disabled-text">Loading more items...</span>
-        <span class="active-text">Show more</span>
+        <span class="active-text"></span>
```

Comment:
Was this change intentional? It's not clear that it's related to the rest of the code in this PR.